app/styles/partials/event-card.scss

```diff
@@ -17,6 +17,20 @@
   }
 }
 
+.extra.content {
+
+  span {
+
+    &:last-child {
```

Comment:
Just use a class, no need for a complicated selector

Reply:
@iamareebjamal Done sir, please check now.